src/include/kdberrors.h

```diff
@@ -0,0 +1,2169 @@
+/*This is an auto-generated file generated by exporterrors. Do not modify it.*/
+
+#ifndef KDBERRORS_H
+#define KDBERRORS_H
+
+#include <kdb.h>
+#include <kdbhelper.h>
+#include <kdblogger.h>
+#include <kdbmacros.h>
+#include <string.h>
+
+#ifdef __cplusplus
+using namespace ckdb;
```

Comment:
@markus2330 I just saw this line... We really should remove that, `using` directives in headers are very much not good practice. I suspect  however, some files already rely on this line being here, so this could be a rather big change.

Reply:
Yes, this definitely need to be removed. Thanks for the hint. I did not look so far into the PR.

I think also some other things will come up as kdberrors.h will be much easier to read in the non-generated way.